source/index.html

```diff
@@ -3,19 +3,26 @@
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Пинк</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
     <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;700&display=swap" rel="stylesheet">
     <link href="css/style.css" rel="stylesheet">
+    <link rel="icon" href="favicon.ico">
```

Comment:
```suggestion
    <link rel="icon" href="/favicon.ico">
```

поменяй урлы на абсолютные

Reply:
у всех, ессно